### PR TITLE
1 Developer Won't Fix, 1 Developer Fixed, 1 Accepted

### PR DESCRIPTION
**Pull request by wtrzas2**
1 test from Incubator-kie-drools marked as DeveloperWon'tFixed per class announcement

1 test from esper repo: `TestSuiteClientCompile.testClientCompileEventTypeAutoName` marked as DeveloperFixed

It was found that on commit `86e64ebccf56e6f9d0fa2d6a4bc0996131c6e229` from Jan 22, 2021 the test failed.
Logs can be found on VM 092: 
```/home/wtrzas2/esper/regression-run/.nondex/V2aNimhf4MzGl3QADJjKYDXHcWwgR7UoJFR1vwsvkk=```

It was found on the next commit `eec1489df9a9c26b5ee27533ddaef48effa0734e` from Jan 22, 2021 the test passed (with 100 NonDex runs).
Logs can be found on VM 092:
```/home/wtrzas2/esper/regression-run/.nondex/Q0efFi4GjrztIZpzxpCA1CbwPAgMEs7MAj9xyBEl+kE=```

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -2737,7 +2737,7 @@ https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1a
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.QueryTest.testQueryWithInheritance,ID,,,
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.QueryTest.testQueryWithUpdateOnFactHandle,ID,,,
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.RuleAttributesTest.testDynamicSalience,ID,,,
-https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.RuleAttributesTest.testSalienceFromConstant,ID,,,
+https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.RuleAttributesTest.testSalienceFromConstant,ID,DeveloperWontFix,,
```
Comment (darko-marinov) [inline pr-data.csv:2740]: Please add the relevant URL in the last column (`Notes`).

Comment (wtrzas2) [reply pr-data.csv:2740]: URL has been added.

**Comment by wtrzas2** [comment]
Also added a changed status from Opened to Accepted for a PR

File: pr-data.csv (hunk 1 above)
Comment (darko-marinov) [reply pr-data.csv:2740]: URL is in the wrong place; it should be in the last column (`Notes`). You may want to change https://github.com/TestingResearchIllinois/idoft/tree/main/format_checker to check for this.

File: pr-data.csv [hunk 2]
```diff
@@ -3479,7 +3479,7 @@ https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810ae
...
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a
-https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,Opened,https://github.com/AxonFramework/AxonFramework/pull/3142,
+https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,Accepted,https://github.com/AxonFramework/AxonFramework/pull/3142,
```
Comment (darko-marinov) [inline pr-data.csv:3482]: It may be better to separate independent changes in different PRs; I could easily accept this change but not others.

File: pr-data.csv (hunk 1 above)
Comment (wtrzas2) [reply pr-data.csv:2740]: Noted, change has been made.

File: pr-data.csv (hunk 2 above)
Comment (wtrzas2) [reply pr-data.csv:3482]: Noted as well, I'll remember for next time.

File: pr-data.csv [hunk 3]
```diff
@@ -3831,7 +3831,7 @@ https://github.com/espertechinc/esper,50eddcdc2e7894de20309cd0b6f585110e4bc61b,c
...
 https://github.com/espertechinc/esper,50eddcdc2e7894de20309cd0b6f585110e4bc61b,common,com.espertech.esper.common.internal.epl.join.assemble.TestLeafAssemblyNode.testProcess,ID,Accepted,https://github.com/espertechinc/esper/pull/226,commit:https://github.com/espertechinc/esper/commit/eec1489df9a9c26b5ee27533ddaef48effa0734e
 https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,examples/rfidassetzone,com.espertech.esper.example.rfidassetzone.TestLRMovingSimMain.testSim,NDOD;NOD,,,
-https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,regression-run,com.espertech.esper.regressionrun.suite.client.TestSuiteClientCompile.testClientCompileEventTypeAutoName,ID,,,
+https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,regression-run,com.espertech.esper.regressionrun.suite.client.TestSuiteClientCompile.testClientCompileEventTypeAutoName,ID,DeveloperFixed,https://github.com/espertechinc/esper/commit/eec1489df9a9c26b5ee27533ddaef48effa0734e,
```
Comment (darko-marinov) [inline pr-data.csv:3834]: Same as previous -- wrong column. The second last column is always for the PR opened by us (so usually `Opened`, `Accepted`, `Rejected` statuses). The last column is for notes (so all other statuses).
